sdk/data/azcosmos/cosmos_location_cache.go

```diff
@@ -5,6 +5,7 @@ package azcosmos
 
 import (
 	"fmt"
+	"log"
```

Comment:
This is fine for ad-hoc testing, but when we actually want to add logging, there's a facility built in to the Core SDK for that in the `"github.com/Azure/azure-sdk-for-go/sdk/internal/log"` package. I used it in a recent commit to the query engine code path: https://github.com/Azure/azure-sdk-for-go/commit/6d8a42734f87cd925e84d39cc942868315445037